### PR TITLE
Agaram.ai Pledge

### PR DESCRIPTION
**Pull request by Agarams**
Agaram.ai ReadMe update for pledge

### CODE REVIEW

File: Agaramai/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Agaram.ai Validator Pledge
+
```
Comment (jeetraut) [inline Agaramai/README.md:2]:
```suggestion
[Agaram.ai Explorer Link](https://main.explorer.agoric.net/validator/agoricvaloper1jjx0vc6dxe44mxzyq2la22j86avdmj0qpkum7f)
```

**Review by jeetraut — CHANGES_REQUESTED** [review]
Add explorer link

File: Agaramai/README.md (hunk 1 above)
Comment (Agarams) [reply Agaramai/README.md:2]: Sure

**Comment by jeetraut** [comment]
Hi. Thanks for the contribution, but we need your README.md file to be in a subfolder. Take a look at https://github.com/Agoric/validator-profiles/pull/2 for an example where the README.md file is BlockShark/README.md.

We could try to resolve the conflicts in this branch / PR, but that would be kind of a hassle, so... can we ask you to just make another PR?

**Pull request closed without merging**